src/network/advanceddispatcher.py

```diff
@@ -57,6 +73,7 @@ def slice_read_buf(self, length=0):
                     del self.read_buf[0:length]
 
     def process(self):
```

Comment:
Process (parse) data that's in the buffer, as long as there is enough data and the connection is open.